### PR TITLE
MEN-4808: tests adjustments for mender CLI commands deprecation

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Tested in https://github.com/mendersoftware/meta-mender/pull/1403

File: tests/test_update.py [hunk 1]
```diff
@@ -78,7 +78,6 @@ class TestUpdates:
     def test_broken_image_update(self, bitbake_variables, connection):
 
         file_flag = Helpers.get_file_flag(bitbake_variables)
-        install_flag = Helpers.get_install_flag(connection)
```
Comment (oleorhagen) [inline tests/test_update.py:81]: :tada:

File: tests/utils/helpers/helpers.py [hunk 2]
```diff
@@ -95,16 +95,6 @@ def get_file_flag(bitbake_variables):
         else:
             return "-u"
 
-    @staticmethod
-    def get_install_flag(connection):
```
Comment (oleorhagen) [inline tests/utils/helpers/helpers.py:99]: Bye :wave:

**Review by oleorhagen — APPROVED** [review]
Thanks

**Pull request merged**